docs/visual-basic/getting-started/additional-resources.md

```diff
@@ -1,88 +1,68 @@
-|[MSDN Chats](http://go.microsoft.com/fwlink/?LinkId=145962)|Provides discussions about Microsoft products or technologies. Each chat is hosted by one or more Microsoft experts. Transcripts are available for completed chats.|  
-  
-### Videos and Webcasts  
-  
-|Term|Definition|  
-|----------|----------------|  
-|[Channel9](http://go.microsoft.com/fwlink/?LinkID=123827)|Provides continuous community through videos, Wikis, and forums.|  
-|[I'm a VB](http://msdn.microsoft.com/vbasic/dd776132)|Watch these interviews and discover what the experts love about Visual Basic and Visual Studio.|  
-  
-### Support  
-  
-|Term|Definition|  
-|----------|----------------|  
-|[Microsoft Help and Support](http://go.microsoft.com/fwlink/?LinkID=108287)|Provides access to Knowledge Base (KB) articles, downloads and updates, support webcasts, and other services.|  
-|[Microsoft Connect](http://connect.microsoft.com/)|Enables you to file bugs or provide suggestions to Microsoft about Visual Studio. You can also report a bug by choosing **Report a Bug** on the **Help** menu.|  
-  
-## Third-Party Resources  
- The MSDN web site provides information on current third-party sites and newsgroups of interest. For the most current list of resources available, see the [MSDN Visual Basic Community Web](http://go.microsoft.com/fwlink/?LinkID=77372) site.  
-  
-|Term|Definition|  
-|----------|----------------|  
-|[DevX Visual Basic Zone](http://go.microsoft.com/fwlink/?LinkId=145978)|Provides in-depth technical articles for today's Visual Basic developer.|  
-|[vb.dotnet.technical](http://go.microsoft.com/fwlink/?LinkId=145986)|Provides a forum for discussion of new features in [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] in the DevX forums.|  
-  
-## See Also  
- [Getting Started](../../visual-basic/getting-started/index.md)  
- [Talk to Us](/visualstudio/ide/talk-to-us)
+# Additional resources
+
+The following web sites provide guidance and can help you find answers to common problems.
+
+## Microsoft resources
+
+### On the web
+
+|Term|Definition|
+|----------|----------------|
+|[Visual Basic .NET Language Design](https://github.com/dotnet/vblang)|Official repository on GitHub for Visual Basic .NET language design.|
+|[Microsoft Visual Basic Team Blog](https://blogs.msdn.microsoft.com/vbteam/)|Provides access to the Visual Basic team blog.|
+
+### Code samples
+
+|Term|Definition|
+|----------|----------------|
+|[Code Gallery](https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=ProgrammingLanguage&f%5B0%5D.Value=VB&f%5B0%5D.Text=VB.NET)|Download and share sample applications and other resources with the developer community.|
+|[Visual Basic documentation samples](https://github.com/dotnet/docs/samples/snippets/visualbasic)|Contains the samples used throughout the Visual Basic and .NET documentation.|
```

Comment:
The link is broken; it should be https://github.com/dotnet/docs/tree/master/samples/snippets/visualbasic